fiftyone/operators/executor.py

```
@@ -828,30 +860,154 @@ def target_view(self, param_name="view_target", require_flat=False):
 
         return sample_collection
 
+    def get_unavailable_view_targets(self, require_flat=False):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)